src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -20,21 +20,82 @@
 import com.google.appengine.api.datastore.PreparedQuery;
 import com.google.appengine.api.datastore.Query;
 import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.gson.Gson;
 import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/**
- * Servlet that returns some example content.
- * comments data
- */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  private final Gson gson = new Gson();
+
+  private final DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+  private static String TIMESTAMP = "timestamp";
+  private static String TITLE = "title";
+  private static String SENTIMENT = "sentiment";
+  private static String UPVOTES = "upvotes";
+  private static String URL = "url";
+  // private static String SUBJECTTAG = "subjecttag";
+  // private static String SUBJECTTAGLIST = "subjecttaglist";
+  // private static String CUSTOMTAGLIST = "customtaglist";
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    // response.sendRedirect("/index.html");
+    /*String settitle = "Orange";
+    int settimestamp = 90;
+    int setupvotes = 32;
+    double setsentiment = -.3;
+    String seturl = "https://google.com";
+    //String setsubjecttag = "";
+
+
+    //Create Enity for the analyzed thread
+    Entity threadEntity = new Entity("Thread");
+    threadEntity.setProperty(TITLE, settitle);
+    threadEntity.setProperty(UPVOTES, setupvotes);
+    threadEntity.setProperty(SENTIMENT, setsentiment);
+    threadEntity.setProperty(TIMESTAMP, settimestamp);
+    threadEntity.setProperty(URL,seturl);
+    //threadEntity.setProperty(SUBJECTTAG, subjecttag);
+
+    datastore.put(threadEntity);
+    Query query = new Query("Thread");*/
+
+    // PreparedQuery results = datastore.prepare(query);
```

Comment:
Don't forget to remove the commented code from the PR: it makes the PR more difficult to understand and could result in unused code / lines being merged unintentionally